content/client-lib-development-guide/features.textile

```diff
@@ -1697,13 +1735,40 @@ class RealtimeChannel:
   unsubscribe(String, (Message) ->) // RTL8a
   setOptions(options: ChannelOptions) => io // RTL16
 
+class BatchOperations:
+  publish([BatchSpec]) => BatchResult<BatchPublishResponse> // BO2a
+  publish(BatchSpec) => BatchResult<BatchPublishResponse> // BO2a
+  getPresence([String]) => BatchResult<BatchPresenceResponse> // BO2b
+
+class BatchResult<T>:
+  error: ErrorInfo? // BPA2b
+  responses: []T? // BPA2a
+
+class BatchPublishResponse:
+  channel: String // BPB2a
+  messageId: String? // BPB2b
+  error: ErrorInfo? // BPB2c
+
+class BatchPresenceResponse:
+  channel: String // BPD2a
+  presence: []BatchPresence // PBD2b
+
+class BatchPresence:
+  clientId: string
+  action: string?
+  error: ErrorInfo?
```

Comment:
I think this error belongs on `BatchPresenceResponse`

Reply:
Good spot, I've moved it over on the new PR in [a33bfbe](https://github.com/ably/docs/pull/1450/commits/a33bfbe04c7b196e1af80824752e352c83294f74) and [384b4fe](https://github.com/ably/docs/pull/1450/commits/384b4fe48a169b3f7b40b839ee25611035a7965b)